src/components/Box/Box.js

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+import { compose, space, color, layout, flexbox } from 'styled-system';
+import { createPropTypes } from '@styled-system/prop-types';
+
+// https://styled-system.com/guides/build-a-box
+
+/** @component */
+export const Box = styled.div`
+  ${compose(space, color, layout, flexbox)}
+
+  // ensures the Box can shrink below its minimum content size when used as a flex item
+  min-width: 0;
+`;
+
+Box.propTypes = {
+  ...createPropTypes(space.propNames),
+  ...createPropTypes(color.propNames),
+  ...createPropTypes(layout.propNames),
+  ...createPropTypes(flexbox.propNames),
+};
+
+export default Box;
```

Comment:
Don't think we need the named export and can just move the `/** @component */` above `export default...`

I tested this out locally and it made the `Warning: src/components/Box/Box.js matches a pattern defined in “components” or “sections” options in your style guide config but doesn’t export a component.` warning dissapear

Reply:
Interesting that I don't get the warning locally. But yes, we can move the docgen comment above the default export to the same effect — I'm still in the habit where older styleguidist required both exports to support outputting the styles and prop type documentation correctly, but that doesn't appear to be an issue now :)